TestObjectiveDropbox/IntegrationTests/TestClasses.m

```diff
@@ -1843,9 +1862,9 @@ - (void)reportsGetStorage:(void (^)(void))nextTest {
 - (void)groupsCreate:(void (^)(void))nextTest {
     [TestFormat printSubTestBegin:NSStringFromSelector(_cmd)];
     [[[_tester.team groupsCreate:_tester.testData.groupName
-               addCreatorAsOwner:@(1)
+               addCreatorAsOwner:@NO
                  groupExternalId:_tester.testData.groupExternalId
-             groupManagementType:nil]
+             groupManagementType:[[DBTEAMCOMMONGroupManagementType alloc] initWithUserManaged]]
       setResponseBlock:^(DBTEAMGroupFullInfo *result, DBTEAMGroupCreateError *routeError, DBRequestError *error) {
```

Comment:
these changes to the tests were needed to handle scoped apps

Reply:
does this change also work with legacy apps? or we don't care legacy apps?

Reply:
i have not tried because we can't make legacy apps anymore unless you're in the gate and API Platform didn't want to add me and told me to use scoped apps.

Reply:
so I think the answer is "we dont care about legacy apps"